.github/workflows/production.yml

```diff
@@ -59,13 +59,32 @@ jobs:
           os=(WINNT_x86_64 Linux_x86_64 Darwin_x86_64-aarch64)
           for j in ${os[@]}
```

Comment:
We should make a note to make the variable names logical here, not entirely sure why I ever decided to set this as j